django/db/models/options.py

```diff
@@ -667,6 +668,28 @@ def get_ancestor_link(self, ancestor):
                 # links
                 return self.parents[parent] or parent_link
 
+    def get_path_to_parent(self, parent):
+        """
+        Get path from current model to a parent model.
```

Comment:
I'm not very familiar with the inner workings of the Django ORM, but it seems to me that the list would contain multiple entries if the chosen parent is a grandparent or more distant model, using multi-table inheritance.